platform/cog-platform-fdo.c

```diff
@@ -427,6 +482,38 @@ static const struct xdg_toplevel_listener xdg_toplevel_listener = {
     .close = xdg_toplevel_on_close,
 };
 
+static void
+xdg_popup_on_configure (void *data,
+                        struct xdg_popup *xdg_popup,
+                        int32_t x,
+                        int32_t y,
+                        int32_t width,
+                        int32_t height)
+{
+    if (popup_data.popup_menu == NULL)
+        return;
+
+    if (!popup_data.configured) {
+        popup_data.configured = true;
+
+        struct wl_buffer *buffer = cog_popup_menu_get_buffer (popup_data.popup_menu);
+        wl_surface_attach (popup_data.wl_surface, buffer, 0, 0);
+        wl_surface_damage (popup_data.wl_surface, 0, 0, INT_MAX, INT_MAX);
```

Comment:
Neat, I didn't know that one can pass sizes bigger than the actual surface size. We should pass `INT32_MAX` here instead of `INT_MAX`, because the parameter is has type `int32_t`, though.